tests/test_requirements.txt

```diff
@@ -16,12 +16,14 @@ pandas==0.23.4
 pytest~=5.2; python_version >= '3.0'
 pytest~=4.6; python_version < '3.0'
 requests==2.20.0
-scipy==1.1.0
+scipy==1.1.0; python_version < '3.0'
```

Comment:
Since Python 2 is not supported, is there a reason to keep this?

Reply:
We're still running tests for py2.7, just not on Travis. It hasn't been dropped yet.